pkg/services/network/nsxt_vpc_provider.go

```diff
@@ -40,6 +40,13 @@ import (
 	"sigs.k8s.io/cluster-api-provider-vsphere/pkg/services"
 )
 
+const (
+	// AnnotationEnableEndpointHealthCheckKey is the key of the annotation that is used to enable health check on the
+	// Service endpoint port. vm-operator propagates annotations in VMService to Service and LB providers like NSX-T
+	// will enable health check on the endpoint target port when this annotation is present on the Service.
+	AnnotationEnableEndpointHealthCheckKey = "lb.iaas.vmware.com/enable-endpoint-health-check"
```

Comment:
Let's move this definition to `constants.go` and rename the key + some adjustments for the godoc:

```suggestion
	// NSXTVPCEnableEndpointHealthCheckAnnotationKey is used by VM Operator to enable health checks for the Service's endpoint port.
	NSXTVPCEnableEndpointHealthCheckAnnotationKey = "lb.iaas.vmware.com/enable-endpoint-health-check"
```

Reply:
Done.